sdcm/report_templates/results_base.html

```diff
@@ -176,6 +176,7 @@ <h3>Test details</h3>
             {% endif %}
             {% if test_id %}
             <li>Test-id: {{ test_id }}</li>
+            <li><a href="https://argus.scylladb.com/tests/scylla-cluster-tests/{{ test_id }}">Link to argus</a></li>
```

Comment:
Typical argus link has `https://argus.scylladb.com/test/%test-id%` structure.
So, are you sure about the `https://argus.scylladb.com/tests/scylla-cluster-tests/{{ test_id }}` ?

Reply:
Forgot to mention that this change would need https://github.com/scylladb/argus/pull/478 to be merged as well to support this link format, but we can start writing them now, they will work once that PR is merged.

Reply:
Added comment there which may influence this PR: https://github.com/scylladb/argus/pull/478#discussion_r1793900485